docs/core/deploying/single-file.md

```diff
@@ -139,18 +139,26 @@ These properties have the following functions:
 Publish a single file application using the [dotnet publish](../tools/dotnet-publish.md) command. When you publish your app, set the following properties:
 
 - Publish for a specific runtime: `-r win-x64`
-- Publish as a single-file: `-p:PublishSingleFile=true`
+- Publish as a single-file: `<PublishSingleFile>true</PublishSingleFile>`
 
 The following example publishes an app for Windows as a self-contained single file application.
 
-```dotnetcli
-dotnet publish -r win-x64 -p:PublishSingleFile=true --self-contained true
+```xml
+<PropertyGroup>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+    <PublishSingleFile>true</PublishSingleFile>
+    <SelfContained>true</SelfContained>
```

Comment:
You don't need to set `SelfContained=true` explicitly. It is defaulted that way when you set `RuntimeIdentifier=foo`.

https://github.com/dotnet/sdk/blob/c53c8ac81e1bcb8118c31bc4f5622396c72e9959/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets#L107-L115